cli/scancommands.go

```diff
@@ -196,7 +196,13 @@ func EnrichCmd(c *components.Context) error {
 func ScanCmd(c *components.Context) error {
```

Comment:
does docker scan uses the same flow as `ScanCmd`? or should implement it there too?

Reply:
No, it is using dockerCmd that resides in CLI. I initiated another PR to ecosystem:
https://github.com/jfrog/jfrog-cli/pull/2686

Reply:
@hadarshjfrog Since resolving the same issue in 'jf docker scan' should be done by Ecosystem and this is not a top priority for them. Since this is not going to happen soon and I dont want to miss the fix Iv'e already worked on im proceeding without 'jf docker scan' and we can open a ticket for the Ecosystem about this